From c4d47b6ea1a1aeab5b8bbdba847147223301e414 Mon Sep 17 00:00:00 2001
From: Adrian Perez de Castro <aperez@igalia.com>
Date: Mon, 17 Oct 2022 11:28:34 +0300
Subject: [PATCH] Add missing header to avoid non-unified build issues

* Source/WebCore/platform/encryptedmedia/CDMInstance.h: Add inclusion of
  the wtf/ThreadSafeRefCounted.h header.
---
 Source/WebCore/platform/encryptedmedia/CDMInstance.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Source/WebCore/platform/encryptedmedia/CDMInstance.h b/Source/WebCore/platform/encryptedmedia/CDMInstance.h
index 7335d42289aa..764281d29e0b 100644
--- a/Source/WebCore/platform/encryptedmedia/CDMInstance.h
+++ b/Source/WebCore/platform/encryptedmedia/CDMInstance.h
@@ -34,6 +34,7 @@
 #include <wtf/Forward.h>
 #include <wtf/Optional.h>
 #include <wtf/RefCounted.h>
+#include <wtf/ThreadSafeRefCounted.h>
 #include <wtf/TypeCasts.h>
 #include <wtf/Vector.h>
 
-- 
2.38.0

